client/tasks/fills/products/product-template-modal.js

```diff
@@ -5,7 +5,7 @@ import { __ } from '@wordpress/i18n';
 import { Button, Modal, RadioControl } from '@wordpress/components';
```

Comment:
Thanks for changing it here as well Fernando 🎉 
Same comments as mentioned in the other `product-template-modal.js` and `products.js` files apply here.